integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSource.scala

```diff
@@ -108,6 +110,10 @@ class CarbonSource extends CreatableRelationProvider
 
     val dbName: String = parameters.getOrElse("dbName", CarbonCommonConstants.DATABASE_DEFAULT_NAME)
     val tableName: String = parameters.getOrElse("tableName", "default_table")
+    if(org.apache.commons.lang.StringUtils.isBlank(tableName) ||
+       org.apache.commons.lang.StringUtils.isWhitespace(tableName)) {
```

Comment:
@jackylk  but what if table name is "a n u b h a v" i think it will not check for its validation